proxies/live/apiproxy/policies/AssignMessage.AccessTokenExpiryOverrideAuthType.xml

```diff
@@ -0,0 +1,7 @@
+<AssignMessage name="AssignMessage.AccessTokenExpiryOverrideAuthType">
```

Comment:
created new policy to use instead of AccessTokenExpiryOverride in refresh token flow and token exchange flow.

AccessTokenExpiryOverride is used in client credentials and auth code flow. 
In these flows no conditional use of nhs login or cis2 kvm values - always use cis2, and the AccessTokenExpiryOverride policy updates identity-service-config.cis2.access_token_expiry_ms directly. 

This new policy can be used in refresh toke and token exchange flow where we use avascript.SetDefaultAccessAndRefreshTokenTimes to set apigee.access_token_expiry_ms time based on whether request is nhs login or cis2

Reply:
though actually looking again- should auth code flow actually not be hardcoded to use cis2?

Reply:
raise a tech debt ticket to combine the 2 policies